smartsim/_core/shell/__init__.py

```diff
@@ -0,0 +1,25 @@
+# BSD 2-Clause License
+#
+# Copyright (c) 2021-2024, Hewlett Packard Enterprise
```

Comment:
Wondering if it would be more consistent for the ``ShellLauncher`` to be in ``_core/launcher/shellLauncher.py`` since DragonLauncher is in ``_core/launcher/DragonLauncher.py``.  Thoughts?

Reply:
Having it under _core/launcher was causing a circular import error!